Pack-Smart/pack-smart-fe
# Refactor/UI cleanup phase 4

## What's this PR do?
Takes Sarah's previous PR and adds to it. Specifically, it:
- Updates Browser favicon, page title and meta text
- Styles thumbnail delete button
- Reverses thumbnails in saved view
- Adds keys to HowItWorks children
- Makes gender button container responsive
- Makes delete modal responsive

## Where should the reviewer start?
- See the code changes

## How should this be manually tested?
- Run `npm start`
- Make sure no there are no longer console errors.
- Check the list above to make sure that all happened

## What are the relevant tickets?
- closes #84 
- closes #82 
- closes #83 
- closes #80 
- closes #76 
- closes #79 



## Did you update the README?
- Nope hehe
